api/scm/sync_org.go

```diff
@@ -137,12 +138,28 @@ func SyncReposForOrg(c *gin.Context) {
 			}
 
 			// update webhook
-			err = scm.FromContext(c).Update(u, repo, lastHook.GetWebhookID())
+			webhookExists, err := scm.FromContext(c).Update(u, repo, lastHook.GetWebhookID())
 			if err != nil {
 				retErr := fmt.Errorf("unable to update repo webhook for %s: %w", repo.GetFullName(), err)
 
 				util.HandleError(c, http.StatusInternalServerError, retErr)
 
+				// if webhook has been manually deleted from GitHub,
+				// set to inactive in database
+				if !webhookExists {
+
+					repo.SetActive(false)
+
+					_, err := database.FromContext(c).UpdateRepo(ctx, repo)
```

Comment:
We'll want a `continue` after the successful execution of this update so we can continue updating repos if one gets tripped up by this `Not Found` business

Reply:
addressed